matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -1478,6 +1478,9 @@ var PhenoTips = (function (PhenoTips) {
             }.bind(this));
 
         }.bind(this));
+
+        //initiate HelpButton widgets
+        Event.fire(document, "xwiki:dom:updated", {elements : this._tableElement.select('.comment-container .xHelpButton')});
```

Comment:
Why don't you use the whole `_tableElement`? The code responsible for `xHelpButton` should know to look inside for it's own trigger elements.